snippets/tests/it/language/c99_tc3.rs

```diff
@@ -0,0 +1,39 @@
+use snippets::{
+    language::c99_tc3, Extractor, Kind, Location, Metadata, Method, Options, PrettyBuffer, Snippet,
+    Transforms,
+};
+use tap::Pipe;
+
+use crate::{assert_snippets_eq, must};
+
+#[test]
+fn full_raw_hello_world() {
+    crate::tracing::setup();
+
+    let content = include_bytes!("testdata/c99_tc3/hello_world.c");
+    let opts = Options::new(Kind::Full, Transforms::none());
+    let extract = c99_tc3::Extractor::extract(&opts, content).expect("must set up parser");
+
+    let expected = vec![Snippet::new(
```

Comment:
This is failing on Windows due to Windows newlines being two bytes  😢

Reply:
Yep, fixed in https://github.com/fossas/foundation-libs/pull/14/commits/07fb1219e32ca6cb94a6aa2c66d486737b94b069!